cli/tests/extensions.rs

```diff
@@ -280,6 +280,135 @@ fn conflicting_extension_name_is_filtered() {
     assert!(output.contains("extension was filtered out"));
 }
 
+mod permissions {
+    use std::ffi::OsStr;
+
+    use assert_cmd::assert::Assert;
+    use predicates::prelude::*;
+
+    use super::*;
+
+    struct TestCLI {
```

Comment:
```suggestion
    struct TestCli {
```

Pretty sure clippy should warn about this too.

Reply:
It does not. I personally prefer this style but I see the official API guidelines also suggest using the `TestCli` form.

Reply:
Yes, I know clippy will warn on this under certain scenarios, might just be because it's a test that it does not. So we should definitely go with `TestCli`.